Polyfills/URL/Source/URL.cpp

```diff
@@ -92,32 +86,27 @@ namespace Babylon::Polyfills::Internal
         return m_searchParamsReference.Value();
     }
 
-    //TODO replace stub with actual implementation
-    Napi::Value URL::CreateObjectURL(const Napi::CallbackInfo& info)
-    {
-        return Napi::Value::From(info.Env(), "");
-    }
-
     URL::URL(const Napi::CallbackInfo& info)
         : Napi::ObjectWrap<URL>{info}
         , m_runtimeScheduler{JsRuntime::GetFromJavaScript(info.Env())}
     {
-        if (!info.Length())
+        if (!info.Length()) {
             return;
+        }
 
         // Store Entire URL
         m_href = info[0].As<Napi::String>();
 
         // Get Position of ? to store search var
         const size_t qIndex = m_href.find_last_of('?');
         
-        if (qIndex != std::string::npos)
+        if (qIndex != std::string::npos) {
             m_search = m_href.substr(qIndex, m_href.size() - qIndex);
-        else
-            m_search = "";
+        }
+
 
         // get UrlSearchParams object 
-        m_searchParams = info.Env().Global().Get(URLSearchParams::JS_URL_SEARCH_PARAMS_CONSTRUCTOR_NAME).As<Napi::Function>().New({Napi::Value::From(info.Env(), m_search) });
+        const Napi::Object m_searchParams = info.Env().Global().Get(URLSearchParams::JS_URL_SEARCH_PARAMS_CONSTRUCTOR_NAME).As<Napi::Function>().New({Napi::Value::From(info.Env(), m_search) });
```

Comment:
Local variables should not have `m_` prefix

```suggestion
        const Napi::Object searchParams = info.Env().Global().Get(URLSearchParams::JS_URL_SEARCH_PARAMS_CONSTRUCTOR_NAME).As<Napi::Function>().New({Napi::Value::From(info.Env(), m_search) });
```